signal2html/html.py

```diff
@@ -80,6 +81,13 @@ def format_message(body, mentions={}):
                 new_body += c
         else:
             new_body += c
+
+    # Make URLs clickable
+    url_regex = "http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+"
```

Comment:
I'm not sure what the purpose of these is: [!*\(\),]? That's rather uncommon. Or did you mean ^ to exclude?
Everything would also work better with r"\b ....... \b"
Add # for the fragment (http://.../x.html#Header)
Since you're not using the groups, we can remove the (non)-captures as well.

Reply:
Thanks for the check! I used the regex suggested in #37, but I should have been more careful in checking it. I'll look into this and update it.

Reply:
Updated url regex in 4db857, please see the extended commit message for more details.

Reply:
Looks ok. I suppose users could add others (new TLDs, other protocols (mailto:, etc.)) by way of a configuration file.

Reply:
Just for your information, I took the original regular expression from the following page, without examining its structure in detail: https://www.w3resource.com/python-exercises/re/python-re-exercise-42.php

Reply:
Probably w3resource took the regular expression from the page called "The Perfect URL Regular Expression": https://urlregex.com/